MCprep_addon/materials/prep.py

```diff
@@ -35,171 +36,158 @@
 # -----------------------------------------------------------------------------
 
 
-class McprepMaterialProps():
-	"""Class to inheret reused MCprep settings.
+class McprepPrepMaterial():
+	"""Base class for Prep Material
 
 	Benefit is to also enforce the same options, required where exec funcitons
 	pass through from one operator to the other (e.g. prep mats on swap packs)
 	"""
 
 	def pack_formats(self, context):
 		"""Blender version-dependant format for cycles/eevee material formats."""
-		itms = []
-		itms.append((
-			"simple", "Simple (no PBR)",
-			"Use a simple shader setup with no PBR or emission falloff."))
-		itms.append(("specular", "Specular", "Sets the pack format to Specular."))
-		itms.append(("seus", "SEUS", "Sets the pack format to SEUS."))
-		return itms
-
-	animateTextures: bpy.props.BoolProperty(
+		return [(i.name.lower(), i.name.capitalize(), i.__doc__) for i in generate.PackFormat]
```

Comment:
This is one of the worst things I've ever seen in Python, and I've seen a lot.

You've taken a perfectly easy to understand function, and compressed it down to this godforsaken mess for what, a really long single line return statement? This barely fits GitHub's file diff, let alone people's editors, and I had to reread this line 10 times to understand what it was doing.

Just no, I'm not allowing this change